api/v1alpha1/etcdcluster_webhook.go

```diff
@@ -256,6 +266,47 @@ func (r *EtcdCluster) validatePdb() (admission.Warnings, field.ErrorList) {
 	return warnings, nil
 }
 
+func (r *EtcdCluster) validateSecurity() field.ErrorList {
+
+	var allErrors field.ErrorList
+
+	if r.Spec.Security == nil {
+		return nil
+	}
+
+	security := r.Spec.Security
+
+	if security.Peer != nil {
+		if (security.Peer.Ca.SecretName != "" && security.Peer.Cert.SecretName == "") ||
+			(security.Peer.Ca.SecretName == "" && security.Peer.Cert.SecretName != "") {
+
+			allErrors = append(allErrors, field.Invalid(
+				field.NewPath("spec", "security", "peer.ca.secretName", "peer.cert.secretName"),
```

Comment:
Does it work correctly?

Reply:
What exactly works correctly and what means correctly? :) I checked and did not have any questions. However we can have a short call and I will show you.

Reply:
I rechecked and found that message was incorrect. See screenshot.
![photo_2024-04-08 13 36 45](https://github.com/aenix-io/etcd-operator/assets/41306070/fb551720-f25b-4cda-8850-af737017b418)

Fixed it.